src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AbsentReasonsDto.java

```diff
@@ -20,4 +14,77 @@
 
     @ApiModelProperty(value = "List of reasons that trigger IEP Warnings", example = "[Refused]")
     private Set<AbsentReason> triggersIEPWarning;
+
+    public AbsentReasonsDto(Set<AbsentReason> paidReasons, Set<AbsentReason> unpaidReasons, Set<AbsentReason> triggersIEPWarning) {
+        this.paidReasons = paidReasons;
+        this.unpaidReasons = unpaidReasons;
+        this.triggersIEPWarning = triggersIEPWarning;
+    }
+
+    public AbsentReasonsDto() {
+    }
+
+    public Set<AbsentReason> getPaidReasons() {
+        return this.paidReasons;
+    }
+
+    public Set<AbsentReason> getUnpaidReasons() {
+        return this.unpaidReasons;
+    }
+
+    public Set<AbsentReason> getTriggersIEPWarning() {
+        return this.triggersIEPWarning;
+    }
+
+    public void setPaidReasons(Set<AbsentReason> paidReasons) {
+        this.paidReasons = paidReasons;
+    }
+
+    public void setUnpaidReasons(Set<AbsentReason> unpaidReasons) {
+        this.unpaidReasons = unpaidReasons;
+    }
+
+    public void setTriggersIEPWarning(Set<AbsentReason> triggersIEPWarning) {
+        this.triggersIEPWarning = triggersIEPWarning;
+    }
+
+    public boolean equals(final Object o) {
```

Comment:
I normally just add a `Getter` lombok annotation to the class and then get Intellij to convert that rather than having to convert the whole `Data` annotation when we don't need to